src/components/Info-Tile.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable react/no-array-index-key */
+import React from 'react';
+import { Link, useStaticQuery, graphql } from 'gatsby';
+import Socials from './Socials';
+
+export default function InfoTile() {
+  const data = useStaticQuery(graphql`
+    query {
+      allMarkdownRemark(sort: {fields: [frontmatter___date], order: DESC}) {
+        edges {
+          node {
+            fields {
+              slug
+            }
+            frontmatter {
+              tags
+              title
+            }
+          }
+        }
+      }
+    }
+`);
+
+  const tags = {};
+  data.allMarkdownRemark.edges.forEach(({ node }) => {
+    node.frontmatter.tags.forEach((tag) => {
+      tags[tag] = tag;
+    });
+  });
+
+  const recentPosts = data.allMarkdownRemark.edges.map(({ node }, index) => {
+    if (index < 10) {
+      return (
+        <Link
+          to={node.fields.slug}
+          key={index}
```

Comment:
This note applies here and to the tagsInJsx below: Always avoid setting `index` for the key in a React list when there's another piece of unique data that would work. 99% of the time it'll work just fine, but sooner or later you'll run into some tricky issues when trying to update/sort/modify the list. A good quick explanation here: https://medium.com/@vraa/why-using-an-index-as-key-in-react-is-probably-a-bad-idea-7543de68b17c

In this case, the best key would be the `node.fields.slug`, since that's guaranteed to be unique, or the `id` of the node (also unique).